DOCUMENTS/POSTFIX_CONF.txt

```diff
@@ -90,12 +98,41 @@ user = postfix
 password = password
 hosts = localhost
 dbname = postfix
-query          = SELECT domain FROM domain WHERE domain='%s' AND active = '1'
+query          = SELECT domain FROM domain WHERE domain='%s' AND active = '1' AND ( primarymx='@@myhostname@@' OR primarymx='' )
 #query          = SELECT domain FROM domain WHERE domain='%s'
 #optional query to use when relaying for backup MX
 #query           = SELECT domain FROM domain WHERE domain='%s' AND backupmx = '0' AND active = '1'
 #expansion_limit = 100
 
+mysql_relay_domains_maps.cf:
+user = postfix
+password = password
+hosts = localhost
+dbname = postfix
+query = SELECT domain FROM domain WHERE domain='%s' AND active = '1' AND primarymx !='@@myhostname@@' AND primarymx!=''
+
+mysql_relay_wildcard_domains_maps.cf:
+user = postfix
+password = password
+hosts = localhost
+dbname = postfix
+query = SELECT '%s' as wcdomain FROM domain WHERE domain=CONCAT('.', domain) AND active='1' AND primarymx !='@@myhostname@@' AND primarymx!=''
+#query = SELECT '%s' as wcdomain FROM domain WHERE domain=CONCAT('.', domain) AND active='1' AND primarymx !='@@myhostname@@' AND primarymx!='' AND backupmx = '1'
+
+mysql_backup_mx_recipients.cf:
+user = postfix
+password = password
+hosts = localhost
+dbname = postfix
+query = SELECT CONCAT('%u', '@', '%d') AS recipient FROM domain WHERE (domain.domain = '%d' OR '%d' LIKE CONCAT('%%.', domain.domain)) AND domain.backupmx = 1
+
+mysql_transport_maps.cf:
+user = postfix
+password = password
+hosts = localhost
+dbname = postfix
+query = SELECT address FROM server JOIN domain ON domain.primarymx = server.server WHERE (domain.domain = '%s' OR CONCAT('.', domain.domain) = '%s') AND domain.primarymx!='@@myhostname@@'
```

Comment:
Shouldn't the 'transport_maps'-query also include ` AND active = '1'`?

FWIW, I love this pull request, and are working on implementing it in my own mail-server setup.